tests/enforcer_test.js

```diff
@@ -180,6 +219,43 @@ describe('TrustedTypesEnforcer', function() {
       expect(el.src).toEqual('');
     });
 
+    it('on Element.prototype.setAttributeNS', function() {
+      let el = document.createElement('iframe');
+
+      expect(function() {
+        el.setAttributeNS('http://www.w3.org/1999/xhtml', 'src', TEST_URL);
+      }).toThrow();
+
+      expect(el.getAttributeNS('http://www.w3.org/1999/xhtml', 'src')).toEqual(null);
+    });
+
+    it('on copy attribute crossing types', function() {
```

Comment:
This test is failing, maybe it's ok to comment it out with TODO fix #47? IIUC we don't have a good idea how to address this yet, but it's not worth blocking the whole PR for that.